zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -160,24 +170,8 @@ private boolean doApproval() throws Exception {
             info().data("user", session.getEmail()).data("collectionId", collection.getDescription().getId())
                     .log("approve task: collection approve task completed successfully");
 
-            // Use only for upload new endpoint
-            if (Configuration.isUploadNewEndpointEnabled()) {
-                // get files here?
-                String fileName = "";
-                for (String string : collectionReader.getReviewed().listUris()) {
-                    if (string.contains("csv") || string.contains("xlsx") || string.contains("csdb") || string.contains("xls")) {
-                        fileName = string.substring(1);
-                        Resource myFile = collectionReader.getResource(fileName);
-                        if (DatasetWhitelistChecker.isWhitelisted(myFile.getName())) {
-                            info().log("File is whitelisted");
-                        } else {
-                            info().log("File is not whitelisted");
-                        }
-                        break;
-                    }
-                }
-            }
-
+            uploadNewEndpoint(collection, collectionReader);
```

Comment:
The feature flag check still needs to go around this - if (Configuration.isUploadNewEndpointEnabled())